src/include/daos/rpc.h

```diff
@@ -93,6 +93,22 @@ enum daos_rpc_type {
 	DAOS_REQ_TGT,
 };
 
+struct daos_req_comm_in {
+	uint32_t	req_in_uid;
+	uint32_t	req_in_gid;
+	uint32_t	req_in_projid;
+	uint64_t	req_in_enqueue_id;
+	uint64_t	req_in_paddings[4];
+	d_string_t	req_in_addr;
```

Comment:
some comments would be great, what is the addr, enqueue_id...

Reply:
Sure.